kiwi/storage/subformat/__init__.py

```diff
@@ -85,7 +86,7 @@ def __new__(self, name, xml_state, root_dir, target_dir):
             return DiskFormatGce(
                 xml_state, root_dir, target_dir, custom_args
             )
-        elif name == 'vmdk':
+        elif name == 'vmdk' or name == 'ova' or name == 'ovf':
             vmdisk_section = xml_state.get_build_type_vmdisk_section()
```

Comment:
I am also wondering if shouldn't we remove the 'ovf' option, since there is no specific code and is treated as an 'ova', I couldn't see any difference (maybe I missed something).

If 'ovf' shouldn't be there thent he schema needs to be updated also.

Reply:
I saw that too but don't wanted to put the burden of a schema change to Michal :) The current schema allows for ova and ovf and I agree with you it should be only ova. I would be ok to do that as a follow up request. The good thing is we don't need a schema version update for this change it would just be one value less of the allowed attr list and the usual auto-code rebuild. But I think it's better we do that as a follow up request. Thoughts ?